File: server/services/usuarios.api/models/classes/boleto.js
class Boleto {
    constructor(boleto) {
        this.no_boleto = boleto.no_boleto || null;
        this.cve_tipo = boleto.cve_tipo || null;
        this.no_operacion = boleto.no_operacion || null;
        this.cve_asiento = boleto.cve_asiento || null;
        this.cve_estado = boleto.cve_estado || null;
        this.nombre_pas = boleto.nombre_pas || null;
        this.token_fac = boleto.token_fac || null;
        this.no_asiento_boleto = boleto.no_asiento_boleto || null;
        this.puerta = boleto.puerta || null;
        this.carril = boleto.carril || null;
        this.anden = boleto.anden || null;
        this.tel_cliente = boleto.tel_cliente || null;
        this.costo_boleto = boleto.costo_boleto || null;
    }
};

module.exports = Boleto;

File: app/Flecha Digital/Model/Boleto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Flecha_Digital.Model
{
    public class Boleto
    {
        public int no_boleto { get; set; }
        public int cve_tipo { get; set; }
        public int no_operacion { get; set; }
        public int cve_asiento { get; set; }
        public int cve_estado { get; set; }
        public string nombre_pas { get; set; }
        public string token_fac { get; set; }
        public int no_asiento_boleto { get; set; }
        public string puerta { get; set; }
        public int carril { get; set; }
        public int anden { get; set; }
        public string metodo_pago { get; set; }
        public string tel_cliente { get; set; }
        public decimal costo_boleto { get; set; }
        public string descripcion_tipo { get; set; }
        public string descripcion_estado { get; set; }
    }
}